### PR TITLE
Feature/test action

### PR DESCRIPTION
**Pull request by domerd**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Test
+
+on:
+  pull_request:
+    branches: [ master ]
```
Comment (yehonatanz) [inline .github/workflows/test.yml]: Why not always? I'd like the CI to run on my branch even before I open a PR

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM domerd/marine-core:marine
+
+RUN python3.8 -m pip install tox
```
Comment (yehonatanz) [inline Dockerfile:3]: How does this image have python3.8?
Shouldn't you `apt install` it just before this `RUN`?

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        key: 'constant-because-docker-handles-cache'
+
+    - name: "Pull marine-core docker image"
+      run: docker pull domerd/marine-core:marine
+
+    - name: "Build marine-python docker image"
+      run: docker build -t marine-python .
+
+    - name: "Run marine-python tests"
+      run: docker run -i --rm -e LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu -v $(pwd)/:/marine-python marine-python
```
Comment (yehonatanz) [inline .github/workflows/test.yml]: Is the `-e LD_LIBRARY_PATH=...` still needed after you properly install marine in `marine-core`?

Comment (yehonatanz) [inline .github/workflows/test.yml]: No need for `/` after `$(pwd)`

File: .dockerignore [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.git
+.tox
+venv
```
Comment (yehonatanz) [inline .dockerignore:3]: Add .venv please :)

**Review by yehonatanz — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (domerd) [reply Dockerfile:3]: ` libglib2.0-dev` installs python3.8 on `marine-core`, therefore it is pre-built.

**Review by yehonatanz — APPROVED** [review]

**Pull request merged**